src/AddIns/Uno.UI.Lottie/ts/Uno.UI.Lottie.ts

```diff
@@ -200,7 +198,7 @@ namespace Uno.UI {
 			element.dispatchEvent(new CustomEvent("lottie_state", { detail: state }));
 		}
 
-		private static getPlayerConfig(properties: LottieAnimationProperties): Lottie.AnimationConfig {
+		private static getPlayerConfig(properties: LottieAnimationProperties, animationData?: AnimationData): Lottie.AnimationConfig {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5477430)